server/api/product_handler.py

```diff
@@ -11,6 +11,7 @@
 
 @product_handler.route('/products/<product_id>', methods=['GET', 'DELETE', 'PUT'])
```

Comment:
I think we also should be adding a ListToProduct connection when adding a product. I think you should be able to add a connection independently, like if you want to follow a product you see on a friend's list, but we'll never be adding a product but not adding it to ListToProduct, right?

Reply:
Correct, I'm going to integrate it so that a POST request is made from the client side to append to the ListToProducts table and to the Products Table